packages/react-instantsearch/src/core/InstantSearch.js

```diff
@@ -29,6 +29,7 @@ function validateNextProps(props, nextProps) {
  * @propType {string} indexName - Main index in which to search.
  * @propType {boolean} [refresh=false] - Flag to activate when the cache needs to be cleared so that the front-end is updated when a change occurs in the index.
  * @propType {object} [algoliaClient] - Provide a custom Algolia client instead of the internal one.
```

Comment:
We can remove this prop from the documentation since we deprecate the option. What is the strategy in the others flavours?

Reply:
Maybe description can have “deprecated” in it